packages/debugger/lib/data/reducers.js

```diff
@@ -310,14 +310,36 @@ function mappedPaths(state = DEFAULT_PATHS, action) {
   }
 }
 
+function allStorageVisible(state = null, action) {
```

Comment:
do these two reducers (and the corresponding actions/selectors) belong in the data app-module? it seems like they should part of evm alongside the actual retrieved storage records.

(also, you mention [here](https://github.com/trufflesuite/truffle/pull/5286/files#diff-c904fc92588b6e51e83081a80402bf454bc2936777a7d010c7249a6507f874b1R225) that this gets set even in light mode, and data normally doesn't run in light mode IIRC)

Reply:
I guess the counterargument is that the data saga is the one that does the orchestration of storage fetching... thus the data module should be responsible for the associated flags.

It's just a bit weird, no? Like, it's related to raw EVM data, not decoded data, _and_ light mode touches it. Seems altogether lower-level.

Reply:
Yeah, it is a bit weird.  You've figured out the reason I put it in data, but it could also go in EVM... and, as you point out, putting it in evm would make the light mode thing less awkward.  I guess I'll go change that.

Reply:
I'd even say that maybe `requestStorage` could be moved to evm altogether... but that would mean you'd have to add arguments to that function, so that data could do the lookups it needs before invoking it. Thoughts?

Reply:
It actually wouldn't require adding arguments, all that info is available in the EVM state.  Note that right now `requestCode` is in data as well... do you think that should be moved too?

Reply:
Oh yeah! Totally! (evm module has access to web3 module?)

Reply:
Yup, should be no problem!